spacepy/toolbox/__init__.py

```diff
@@ -1880,27 +1913,27 @@ def logspace(min, max, num, **kwargs):
         number of log spaced bins
 
     Other Parameters
-    ================
+    ----------------
     kwargs : dict
         additional keywords passed into matplotlib.dates.num2date
 
     Returns
-    =======
+    -------
     out : array
         log-spaced bins from min to max in a numpy array
 
     Notes
-    =====
-    This function works on both numbers and datetime objects
+    -----
+    This function works on both numbers and datetime objects. This function is not leap second aware.
```

Comment:
Seems obvious since datetime isn't leapsecond aware, but I presume the idea is that the spacing is wrong in an absolute system (e.g. 24 bins from 2016-12-31T00 through 2017-01-01 will be 3600s apart not 3600.04 in the linspace case). Might be worded more clearly?

Total nitpick: Would just put "Works on both numbers and datetime objects. Not leapsecond aware." Sentence fragments seem fine.

Reply:
Updated, I think we need to be explicit here as if it fell through to `time` it could be leapsecond aware.